folding-schemes/src/utils/mod.rs

```diff
@@ -21,3 +23,12 @@ pub fn powers_of<F: PrimeField>(x: F, n: usize) -> Vec<F> {
     }
     c
 }
+
+/// returns the coordinates of a commitment point. This is compatible with the arkworks
+/// GC.to_constraint_field()[..2]
+pub fn get_cm_coordinates<C: CurveGroup>(cm: &C) -> Vec<C::BaseField> {
```

Comment:
`pub(crate)` maybe?

Reply:
well, probably any method in the `utils` folder and subfolders would be `pub(crate)`, but I'm unsure about it because then at somepoint somebody trying to do further stuff with the lib may require some of those methods and will do PRs to make them public again (happened in other libs), so I'm not sure on what's best here but until now I was approaching it with the mindset of allowing others to hack with 'internal' utils of this library. How do you see it?

Reply:
I'm cool with it as long as only the `utils` module is exposed. Such that the autocompletion and API discovery doesn't get super polluted with `utils`-related stuff.